resources/views/faq.blade.php

```diff
@@ -12,7 +12,7 @@
             <dl class="grid-cols-1 gap-y-5 md:gap-x-10 md:gap-y-16 md:grid-cols-2 grid">
                 <div class="border rounded border-gray-300 shadow p-4 dark:bg-gray-800 dark:border-gray-700">
                     <dt class="leading-7 font-semibold text-base dark:text-white">What is the “We are in IT together Conference”?</dt>
-                    <dd class="leading-7 mt-2 dark:text-gray-200">The 'We are in IT together Conference' is an inclusive platform where students, teachers, and IT company representatives collaborate through workshops, presentations, and booths. This event promotes knowledge exchange, enabling students to explore the dynamic IT industry and providing companies opportunities to connect with potential employees, offer internships, and engage with the university's educational approach.</dd>
+                    <dd class="leading-7 mt-2 dark:text-gray-200">The 'We are in IT together Conference' is an inclusive platform where students, teachers, and IT company representatives collaborate through workshops, presentations, and booths. This event promotes knowledge exchange, enabling students to explore the dynamic IT industry and providing companies opportunities to connect with potential employees, offer internships, and engage with the university's educational approach. If you want to know more, check out our <a href="https://www.youtube.com/@WeareInITTogether" target="_blank" class="text-yellow-300 hover:text-yellow-500 hover:border-b-2 hover:border-yellow-500 transition-all">YouTube channel</a> and last year's <a href="https://www.youtube.com/watch?v=k-Psaz0hdQI" target="_blank" class="text-yellow-300 hover:text-yellow-500 hover:border-b-2 hover:border-yellow-500 transition-all">impression video</a> about the Conference.</dd>
```

Comment:
UI/UX: The colors currently in use for the links are not that readable, it doesn't meet accessibility standards. I would suggest using the colors that are used for the other link on this page:

`text-blue-600 hover:text-blue-400 visited:text-purple-600`